### DISCUSSION TITLE
Any way to disable activity events?

### DISCUSSION THREAD

**Discussion by ejsmith** [open-telemetry/opentelemetry-dotnet#3402] [General]
Elasticsearch APM turns activity events into log messages and it's creating incredible amounts of noise in my logs. Is there a way to turn off reporting activity events or to be able to clear the events before activities are submitted?

**Comment by cijothomas** [comment]
The Exporters has no turn-off-events switch today. You may want to stop adding activity in the 1st place, if not planning to export it. Are you affected by https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/416 ? or a separate instrumentations?

**Reply by ejsmith** [reply]
@cijothomas that is exactly where my problems are coming from and I was going to submit the exact same PR. :-)
